### DISCUSSION TITLE
Bitbucket Cloud and Pull Requests

### DISCUSSION THREAD

**Discussion by EstebanMonge** [jenkinsci/bitbucket-push-and-pull-request-plugin#216] [Q&A]
Hello, I am using Bitbucket Cloud, I want to build the master branch with a PR merged locally, but Cloud doesn't support refs. How can I build the master repo with a PR merged locally?

**Comment by EstebanMonge** [comment]
I have read about the theme I make a api call to get the url from the original repo to the fork repo. This is the unique way to get the new PR tested.

**Comment by cdelmonte-zg** [comment]
@EstebanMonge Maybe is this link interesting for you?
 [How do I try out a pull request before merging it?](https://community.atlassian.com/t5/Answers-Developer-Questions/How-do-I-try-out-a-pull-request-before-merging-it/qaq-p/463050)

**Comment by EstebanMonge** [comment]
Thanks @cdelmonte-zg, that works with Bitbucket Server, but with Cloud doesn't works
